trikotFoundation/src/iosArm32Main/kotlin/com/mirego/trikot/foundation/concurrent/dispatchQueue/iOSGlobalDispatchQueue.kt

```diff
@@ -0,0 +1,22 @@
+package com.mirego.trikot.foundation.concurrent.dispatchQueue
+
+import com.mirego.trikot.foundation.concurrent.freeze
+import platform.darwin.DISPATCH_QUEUE_PRIORITY_HIGH
+import platform.darwin.dispatch_async
+import platform.darwin.dispatch_get_global_queue
+
+open class iOSGlobalDispatchQueue : DispatchQueue {
```

Comment:
`IOSGlobalDispatchQueue`

Reply:
What's different about arm32 that we can't use the same code? The long types?

Reply:
Exactly, the long flag variable 🙄